.github/actions/codacy/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+CODACY_TOKEN=$1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/manual/davidmerrick/AdventOfCode2019/pullRequest?prid=5020686)